.codespellrc

```diff
@@ -0,0 +1,4 @@
+[codespell]
+# skipping auto generated folders
+skip = ./.tox,./.mypy_cache,./docs/_build,./target
+ignore-words-list = ot,soket
```

Comment:
added `soket` since I didn't want to change the code without discussing with approvers first just for the sake of spellcheck

Reply:
It's fine. You can update it's refs.